routes/api.php

```diff
@@ -10,6 +11,26 @@
 use App\Models\Label;
 use App\Http\Controllers\AddImageController;
 
+function cmsQuery(Request $request, Builder $query) {
+    $validatedData = $request->validate([
+        'limit' => 'integer|min:0',
+        'sort' => 'in:provisioning_started_at'
+    ]);
+
+    $limit = $validatedData['limit'] ?? null;
+    $sort = $validatedData['sort'] ?? null;
+
+    if ($limit) {
+        $query = $query->take($limit);
+    }
+    if ($sort) {
+        $query = $query->orderBy($sort);
```

Comment:
Should this be descending so that it returns the most recent entries by default?